### DISCUSSION TITLE
Are there any external controllable usb Fan Controller known?

### DISCUSSION THREAD

**Discussion by menotuu** [Rem0o/FanControl.Releases#2052] [Q&A]
Please don't delete.

I like to know from the big community:

Is there anything like external fan controls that can be connected via USB (Maybe with mosfets/relays) and controlled by this fan control Software?

Because I try to control a Laptop Fan, for that no Fan control exist, but I can solder... So a easier way for me if such exist.
Lenovo Thinkbook 14 G4. (Yes I tried all)

**Comment by Rem0o** [comment]
Moved from issue to discussion.

**Comment by RafhaanShah** [comment]
AquaComputer Octo is reported to work
